cpp/ql/src/Likely Bugs/Likely Typos/FutileParams.ql

```diff
@@ -0,0 +1,18 @@
+/**
+ * @name Non-empty call to function declared without parameters
+ * @description A call to a function declared without parameters has arguments, which may indicate
+ *              that the code does not follow the author's intent.
+ * @kind problem
+ * @problem.severity warning
```

Comment:
From the results you've shown so far, I think we can make it at least `@precision high`.